### DISCUSSION TITLE
Can we do anomaly detection on grayscale images?

### DISCUSSION THREAD

**Discussion by harshsinha03** [open-edge-platform/anomalib#1998] [Q&A]
Is it possible to train models on grayscale images?

**Comment by abc-125** [comment]
Theoretically, it should be, but it would require modifying the feature extractor for all the models that use it (most of the models, I think). You can find details on how to do it and if it's worth doing it for classification tasks [here](https://towardsdatascience.com/transfer-learning-on-greyscale-images-how-to-fine-tune-pretrained-models-on-black-and-white-9a5150755c7a).
